UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/LogStructure.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
+{
+    [AddComponentMenu("Scripts/MRTK/Examples/LogStructure")]
```

Comment:
Please remove the `Demos.EyeTracking` from the namespace

Nit Pick/Style....Please start fields with a lowercase letter, not underscore.